tensorflow/lite/micro/tools/generate_cc_arrays.py

```diff
@@ -73,11 +74,13 @@ def generate_array(input_fname):
     return [len(image_bytes), out_string]
   elif input_fname.endswith('.wav'):
     wav_file = wave.open(input_fname, mode='r')
+    num_channels = wav_file.getnchannels()
     out_string = ''
     for _ in range(wav_file.getnframes()):
       frame = wav_file.readframes(1)
-      out_string += str(int.from_bytes(frame, byteorder='little',
-                                       signed=True)) + ','
+      samples = struct.unpack('<%dh' % num_channels, frame)
+      for sample in samples:
+        out_string += str(sample) + ','
```

Comment:
This is a comment about the existing code, but it can be inefficient (https://google.github.io/styleguide/pyguide.html#310-strings)

Can we change to:
```py
n_frames = wav_file.getnframes()
frames = wav_file.readframes(n_frames)
samples = struct.unpack('<%dh' % (num_channels * n_frames), frames)
out_string = ','.join(map(str, samples))
```